spring-cloud-sleuth-core/src/test/java/org/springframework/cloud/sleuth/instrument/web/SpringDataInstrumentationTests.java

```diff
@@ -45,7 +45,7 @@
 import org.springframework.core.env.Environment;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
-import org.springframework.hateoas.PagedResources;
+import org.springframework.hateoas.PagedModel;
```

Comment:
@shakuzen  This change is unrelated but fixes the compilation issue on master (due to Spring HATEAOS upgrade), please let me know if you want me to revert or keep it (my only intention was to see the CI build status).

Reply:
@marcingrzejszczak will probably be happy to accept a fix :)